llvm/lib/Analysis/ScalarEvolution.cpp

```diff
@@ -2981,7 +2981,7 @@ ScalarEvolution::getOrCreateAddExpr(ArrayRef<const SCEV *> Ops,
       static_cast<SCEVAddExpr *>(UniqueSCEVs.FindNodeOrInsertPos(ID, IP));
   if (!S) {
     const SCEV **O = SCEVAllocator.Allocate<const SCEV *>(Ops.size());
-    std::uninitialized_copy(Ops.begin(), Ops.end(), O);
+    llvm::uninitialized_copy(Ops, O);
```

Comment:
Do you need the `llvm::`? Most .cpp files in LLVM have a `using namespace llvm;`.

Reply:
AFAICT, it seems to be leaning towards using `llvm::` for variants of std::, so I am keeping the llvm:: for now.

Reply:
Resolving.